embassy-stm32/build.rs

```diff
@@ -508,6 +510,16 @@ fn main() {
                     let field_name = format_ident!("{}", field_name);
                     let enum_name = format_ident!("{}", enum_name);
 
+                    if !rcc_cfgr_regs.contains_key(mux.register) {
+                        rcc_cfgr_regs.insert(mux.register, Vec::new());
+                    }
+
+                    rcc_cfgr_regs.get_mut(mux.register).unwrap().push((
```

Comment:
you can do this with `.entry(..).or_insert_with(..)`. Shorter and avoids the unwrap.